procrastinate/aiopg_connector.py

```diff
@@ -1,26 +1,70 @@
-from typing import Any, Awaitable, Dict, Optional
+import asyncio
+from typing import Any, Awaitable, Dict, List, Optional
 
 import aiopg
-from psycopg2.extras import RealDictCursor
+import psycopg2.sql
+from psycopg2.extras import Json, RealDictCursor
 
-from procrastinate import psycopg2_connector, store
+from procrastinate import store
 
 
-def get_connection(**kwargs) -> Awaitable[aiopg.Connection]:
-    return aiopg.connect(**kwargs)
+def wrap_json(arguments: Dict[str, Any]):
+    return {
+        key: Json(value) if isinstance(value, dict) else value
+        for key, value in arguments.items()
+    }
+
+
+def get_connection(dsn="", **kwargs) -> Awaitable[aiopg.Connection]:
+    return aiopg.connect(dsn=dsn, **kwargs)
+
+
+async def execute_query(
+    connection: aiopg.Connection, query: str, **arguments: Any
+) -> None:
+    # Strangely, aiopg can work with psycopg2's cursor class.
```

Comment:
aiopg is just a layer on top of psycopg2, this isn't that weird ;)

Reply:
Yeah, but it's quite the abstraction leak... Having an `import psycopg2` in a project that doesn't define psycopg2 as a dependency...